utensor_cgen/frontend/tflite.py

```diff
@@ -79,7 +79,7 @@ def _build_tensor_map(self, fb_model, ugraph):
 
     for idx in range(0, subgraph.TensorsLength()):
       tensor = subgraph.Tensors(idx)
-      tensor_name = tensor.Name().decode('utf8')
+      tensor_name = tensor.Name().decode('utf8').replace(';', '__')
```

Comment:
Do you know if there are any other possible corner cases? It might help to display a warning here if we run into an odd literal in parsing the tensor names

Reply:
@dboyliao bump

Reply:
Honestly, I don't know.
I really don't want to dig deep into tflite converter's C code for the tensor name issue here.
What kind of warning?
like this?
```python
if ';' in tensor.Name():
    # warning code here
    .... do stuff
```

Reply:
Yeah, just something that warns “Unexpected character, %s, found in tensor name”

Reply:
Ok, done.
<img width="1117" alt="Screen Shot 2020-10-28 at 10 07 50 PM" src="https://user-images.githubusercontent.com/6830390/97447172-3f0d5c00-196a-11eb-8708-cbf39a48ba94.png">

I'll merge this PR, ASAP